improver_tests/acceptance/test_spot_extract.py

```diff
@@ -145,7 +145,7 @@ def test_lapse_rate_mismatch(tmp_path):
     neighbour_path = kgo_dir / "inputs/all_methods_uk.nc"
     diag_path = kgo_dir / "inputs/ukvx_temperature.nc"
     lapse_path = kgo_dir / "inputs/ukvx_lapse_rate_2m.nc"
-    kgo_path = kgo_dir / "outputs/nearest_uk_temperatures.nc"
+    kgo_path = kgo_dir / "outputs/postprocessed_nearest_uk_temperatures.nc"
```

Comment:
This change is interesting. The test is for a case in which the lapse rate is not actually applied because the lapse rate and temperature cubes have mismatched height coordinates. However, by virtue of going through the plugin the title is updated even though the data is not.

I wonder whether the warning in the plugin ought to be an exception? That would make the user aware of the mismatch more starkly than the warning which allows everything to chug along. It would also avoid this issue in which we can send cubes in and out of the plugin whilst not actually doing anything to them.

Thoughts?

Reply:
I can't think of a good reason why we don't want an Exception in this case. I wonder if an early iteration of the suite required this flexibility.

Reply:
I've updated this be an exception now